### PR TITLE
feat: display attribute types on input fields

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

- Displays attribute types on input fields
- Also includes some changes to the documents table, after talking with design

## Test Plan

Manual, plus tests for two new helper functions

## Related PRs and Issues

- #216 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/lib/helpers/string.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Capitalizes the first letter of a string
+ *
+ * @export
+ * @param {string} str
+ * @returns {string}
+ */
+export function capitalize(str: string): string {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
```
Comment (TorstenDittmann) [inline src/lib/helpers/string.ts:10]: It might be smarter to use CSS for this 🤔

Comment (TGlide) [reply src/lib/helpers/string.ts:10]: The behaviour is different. With CSS, the first letter of every word is capitalized. This function onnly does it for the first word. The question is, which one is preferred?

Comment (TorstenDittmann) [reply src/lib/helpers/string.ts:10]: iirc there should be a `:first-letter` selector 👀

Comment (TGlide) [reply src/lib/helpers/string.ts:10]: Oh, TIL! Will use CSS then, for sure :smile:

Comment (TGlide) [reply src/lib/helpers/string.ts:10]: @TorstenDittmann After further inspection, the way I see it, in order to use CSS I'd have to either pass in a custom prop to the inputs being used to Attribute, or use :global() in AttributeForm's styles. With this in mind, I think we're better off keeping this as is. What do you think?

**Comment by TorstenDittmann** [comment]
Bildschirm­foto 2023-01-27 um 13 30 39

Code wise looks good! But there is a small issue with the event text input 👀

File: src/lib/helpers/string.ts (hunk 1 above)
Comment (TorstenDittmann) [reply src/lib/helpers/string.ts:10]: nah, wanted to make it simpler - this is not the way :-D

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**